src/app.js

```diff
@@ -0,0 +1,34 @@
+// const express = require("express");
+import express from "express";
+import morgan from "morgan";
+import mongoose from "mongoose";
+import "dotenv/config";
+import routes from "./routes";
+import passport from "passport";
+import authRoutes from './routes/api/authRoutes'
+
+const secureRoute = "./secure-routes";
+
+const app = express();
+app.use(express.json());
+app.use(morgan("dev"));
+
+
+const port = process.env.PORT || 4000;
+
+
+try {
+ 
+  mongoose.connect("mongodb://localhost:27017/marthadb", { useNewUrlParser: true });
```

Comment:
@Marthatwesi do not expose your database URL to the public. Use environment variables.
Also, remove unnecessary empty lines.